ignite/services/chain/chain.go

```diff
@@ -424,14 +419,14 @@ func (c *Chain) Commands(ctx context.Context) (chaincmdrunner.Runner, error) {
 		return chaincmdrunner.Runner{}, err
 	}
 
-	config, err := c.Config()
+	cfg, err := c.Config()
 	if err != nil {
 		return chaincmdrunner.Runner{}, err
 	}
 
 	servers := chainconfigv1.DefaultServers()
-	if len(config.Validators) > 0 {
-		validator := config.Validators[0]
+	if len(cfg.Validators) > 0 {
+		validator := cfg.Validators[0]
```

Comment:
Shouldn't we use `FirstValidator()` here as well?

Reply:
IMO we should check it for consistency.

Reply:
Changeset 6f34c37